defaults.yml

```diff
@@ -962,6 +962,26 @@ categories:
         after: killall Activity\ Monitor
         versions: [Monterey]
 
+  - folder: general
+    name: General
+    description: |
+      On a Mac, there are many General settings which are able to be set.
+    keys:
+      - key: NSCloseAlwaysConfirmsChanges
+        domain: .GlobalPreferences.plist
+        title: Close Always Confirm Changes
+        description:
+          Should you be asked to keep changes when closing documents or just have changes saved automatically?
+        param:
+          type: int
+        examples:
+          - value: 0
+            default: 0
+            text: Auto saving is enabled, you are not prompted to save changes.
+          - value: 2
+            text: Auto saving is disabled, you are prompted if you want to save changes.
```

Comment:
Why not use a boolean? I think it's more clear with "true" or "false".

Maybe @yannbertrand  has a preference?

Reply:
For this value, I am actually not sure if boolean will work. Would require some testing. I agree true or false is far more clear. But I would have guessed that would use values 0 and 1. Not sure why but when you tick it in the system preferences it sets to integer 2.

Reply:
Perhaps there is a third state between these. I have not investigated any further.